render_pdf/src/make_latex.py

```diff
@@ -0,0 +1,92 @@
+from typing import Union
+import parser
+import os
+import jinja2
+
+# TODO probably best to use pydantic for a config template for the dict below. 
+data = {
+        'language' : "sanskrit",
+        'title' : "",
+        'author' : "",
+        'body' : [],
+        'analysis': []
+    }
+
+os.chdir("../tex_files")
+latex_jinja_env = jinja2.Environment(
+        block_start_string = '((*',
+        block_end_string = '*))',
+        variable_start_string = '(((',
+        variable_end_string = ')))',
+        comment_start_string = '((=',
+        comment_end_string = '=))',
+        loader = jinja2.FileSystemLoader(os.path.abspath('.'))
```

Comment:
`pathlib` is much nicer, grep around in the `ambuda` folder for examples.

Reply:
I'll leave this as is since I don't want to spend time with pathlib etc. I want to get something out, and I don't think this is going to prevent the product from working properly. I've already moved to taking an input file as a sys.argv. But I didn't use an output file as input though. I have the output names standardized contingent on the metadata. 

We can revisit this if the calling script for this module needs to be able to assign names.